hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -763,9 +764,11 @@ public synchronized void close() throws IOException {
               tracingHeaderFormat,
               listener, abfsCounters.toString());
       try {
-        getAbfsClient().getMetricCall(tracingMetricContext);
+        if (!tracingMetricContext.getMetricResults().equals(EMPTY_STRING)) {
```

Comment:
Nit: Can be simplified for OSS as follow?
```
if (abfsStore.getClient().isMetricCollectionEnabled() && !abfsCounters.toString().equals(EMPTY_STRING)) {
      try {
        TracingContext tracingMetricContext = new TracingContext(
            clientCorrelationId,
            fileSystemId, FSOperationType.GET_ATTR, true,
            tracingHeaderFormat,
            listener, abfsCounters.toString());
        getAbfsClient().getMetricCall(tracingMetricContext);
      } catch (IOException e) {
        LOG.error("Error while getting metrics from client", e);
      }
 }
```

Reply:
Taken